htdocs/themes/berlin_mcjc/sass/_screen.scss

```diff
@@ -930,323 +1016,350 @@ ul.title-list {
-        border: 1px solid #ccc;
-        padding: 2px;
-    }
+  a {
+    display: block;
+    height: rhythm(4);
+    border: 1px solid #ccc;
+    padding: 2px;
+  }
 
-    img {
-        height: 100%;
-        width: auto;
-    }
+  img {
+    height: 100%;
+    width: auto;
+  }
 }
 
 .collections .item {
-    overflow: hidden;
-    border-color: #ccc;
-    @include leading-border(1px, 0, $base-font-size, solid);
-    @include leader();
+  overflow: hidden;
+  border-color: #ccc;
+  @include leading-border(1px, 0, $base-font-size, solid);
+  @include leader();
 }
 
 /* @end */
 
 /* @group ----- Search Results ----- */
 
-#search-results + .pagination { @include leader(1.5); }
+#search-results + .pagination {
+  @include leader(1.5);
+}
 
 #search-results td {
-    vertical-align: top;
+  vertical-align: top;
 }
 
 #search-results .image {
-    float: left;
-    height: rhythm(4);
-    margin: 0 rhythm() 0 0;
+  float: left;
+  height: rhythm(4);
+  margin: 0 rhythm() 0 0;
 
-    img {
-        height: 100%;
-        width: auto;
-    }
+  img {
+    height: 100%;
+    width: auto;
+  }
 }
 
-#item-filters ul, #search-filters > ul {
-    padding-left: 0;
-    list-style-type: none;
-    border-bottom: 1px solid #eee;
-    margin: rhythm(-.5) 0 0;
-
-    & > li {
-        display: inline-block;
-        background-color: #eee;
-        padding: rhythm(.25) rhythm(.5);
-        margin: 0 rhythm(.5) rhythm(.5) 0;
-    }
+#item-filters ul,
+#search-filters > ul {
+  padding-left: 0;
+  list-style-type: none;
+  border-bottom: 1px solid #eee;
+  margin: rhythm(-0.5) 0 0;
 
-    & > li:last-child:after { content: "\00A0"; }
+  & > li {
+    display: inline-block;
+    background-color: #eee;
+    padding: rhythm(0.25) rhythm(0.5);
+    margin: 0 rhythm(0.5) rhythm(0.5) 0;
+  }
 
-    li ul { display:inline; }
+  & > li:last-child:after {
+    content: "\00A0";
+  }
+
+  li ul {
+    display: inline;
+  }
 
-    li ul li { display:inline-block; }
+  li ul li {
+    display: inline-block;
+  }
 
-    li ul li:after {
-        content: ", ";
-        margin: 0;
-    }
+  li ul li:after {
+    content: ", ";
+    margin: 0;
+  }
 
-    li ul li:last-child:after { content: ""; }
+  li ul li:last-child:after {
+    content: "";
+  }
 }
 
 /* @end */
 
 /* @group ----- Footer ----- */
 
 footer {
-    clear:both;
-    @include container;
-    overflow:hidden;
-    @include leader();
-
-    a { text-decoration:none; }
-
-    .navigation {
-        font-weight:bold;
+  clear: both;
+  @include container;
+  overflow: hidden;
+  @include leader();
+  background-color: #eaeaea;
+
+  a {
+    text-decoration: none;
+  }
 
-        li {
-            display:inline;
+  .navigation {
+    font-weight: bold;
 
-            &:after {
-                content: " \00B7";
-                margin: 0 10px 0 5px;
-            }
+    li {
+      display: inline;
 
-            &:last-child { margin: 0; }
+      &:after {
+        content: " \00B7";
```

Comment:
This rule is causing the footer nav to have these extraneous dots in it -- wondering if it can be removed completely or if it's needed elsewhere?